tests/backup/backup_delete_test.go

```diff
@@ -468,3 +469,226 @@ var _ = Describe("{DeleteIncrementalBackupsAndRecreateNew}", func() {
+			clusterUid, err = Inst().Backup.GetClusterUID(ctx, orgID, SourceClusterName)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching [%s] cluster uid", SourceClusterName))
+		})
+
+		Step("Creating schedule backup", func() {
+			log.InfoD("Creating schedule backup")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			periodicSchedulePolicyUid, _ = Inst().Backup.GetSchedulePolicyUid(orgID, ctx, periodicSchedulePolicyName)
+			for _, namespace := range appNamespaces {
+				scheduleName = fmt.Sprintf("%s-schedule-%v", BackupNamePrefix, time.Now().Unix())
+				scheduleNames = append(scheduleNames, scheduleName)
+				labelSelectors := make(map[string]string)
+				err = CreateScheduleBackup(scheduleName, SourceClusterName, bkpLocationName, backupLocationUID, []string{namespace},
+					labelSelectors, orgID, "", "", "", "", periodicSchedulePolicyName, periodicSchedulePolicyUid, ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verification of creating schedule backup with schedule name - %s", scheduleName))
+				schBackupName, err := GetFirstScheduleBackupName(ctx, scheduleName, orgID)
+				log.FailOnError(err, "Getting first schedule backup name")
+				backupNames = append(backupNames, schBackupName)
+			}
+		})
+
+		Step("Creating a manual backup", func() {
+			log.InfoD("Creating a manual backup")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, namespace := range appNamespaces {
+				backupName := fmt.Sprintf("%s-%v", BackupNamePrefix, time.Now().Unix())
+				scheduleNames = append(scheduleNames, scheduleName)
+				labelSelectors := make(map[string]string)
+				err = CreateBackup(backupName, SourceClusterName, bkpLocationName, backupLocationUID, []string{namespace}, labelSelectors, orgID, clusterUid, "", "", "", "", ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying backup %s creation", backupName))
+				backupNames = append(backupNames, backupName)
+			}
+		})
+
+		Step("Suspending the existing backup schedules", func() {
+			log.InfoD("Suspending the existing backup schedules")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Fetching px-central-admin ctx")
+			for _, scheduleName := range scheduleNames {
+				scheduleUid, err := GetScheduleUID(scheduleName, orgID, ctx)
+				log.FailOnError(err, "Error while getting schedule uid %v", scheduleName)
+				err = suspendBackupSchedule(scheduleName, scheduleUid, periodicSchedulePolicyName, periodicSchedulePolicyUid, orgID, ctx)
+				dash.VerifySafely(err, nil, fmt.Sprintf("Verification of suspending backup schedule - %s", scheduleName))
+			}
+		})
+
+		Step("Delete the bucket where the backup objects are present", func() {
+			log.InfoD("Delete the bucket where the backup objects are present")
+			for _, provider := range providers {
+				globalBucketName := getGlobalBucketName(provider)
```

Comment:
Why play around with global bucket can we use test specific bucket instead?

Reply:
Made changes to use local bucket for testcase